trust-quorum/src/connection_manager.rs

```diff
@@ -446,19 +446,16 @@ impl ConnMgr {
             );
 
             // Inform the main task that accepted connection is established
-            if let Err(e) = main_tx
-                .send(ConnToMainMsg {
-                    task_id: task::id(),
-                    msg: ConnToMainMsgInner::Accepted {
-                        addr,
-                        peer_id: baseboard_id,
-                    },
-                })
-                .await
-            {
-                // The system is shutting down
+            if let Err(_) = main_tx.try_send(ConnToMainMsg {
+                task_id: task::id(),
+                msg: ConnToMainMsgInner::Accepted {
+                    addr,
+                    peer_id: baseboard_id,
+                },
+            }) {
+                // The system is shutting down or we've overloaded the main channel
                 // Just bail from this task
-                warn!(log, "Failed to send 'accepted' msg to main task: {e:?}");
+                warn!(log, "Failed to send 'accepted' msg to main task");
```

Comment:
now that we're using `try_send`, i think it's actually _more_ important to log which error was encountered --- i realize that the `{e:?}` was removed since `TrySendError` actually returns the message type, which may not implement `Debug` here. but, unlike the previous code which used `send`, an error may be returned either because the main task has terminated *or* because the channel is full, which are different enough situations that I think we should probably indicate why in the log.

maybe consider matching the error type and logging a different message?